### DISCUSSION TITLE
Unable to reverify my account as a teacher

### DISCUSSION THREAD

**Discussion by ssp3nc3r** [community/community#49950] [GitHub Education]
Hello. 

I've tried numerous times to reverify my account as a teacher. And I've submitted an issue with Github about is and all they do is close the ticket ([Ticket 2030656](https://support.github.com/ticket/2030656))) and refer me to the gitub education document and FAQ.

In the latest attempt, I received the following email:

> Hey ssp3nc3r,
> 
> Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Mar 13, 2023 for [@ssp3nc3r](https://github.com/ssp3nc3r).
> 
> You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
> If you meant to apply as a student, please cancel your application and reapply [here](https://education.github.com/discount_requests/student_application).
> 
> Once you have resolved the issues with your sumbission, [visit us again and reapply](https://education.github.com/).

But I do have my full name on my github account, have identified my school (Columbia University), and even have a link to my faculty page.

And I've tried submitting a photo of my school id both using the take a photo button, and using the upload a file button. It has my name, school name, expiration date, and a photo. 

Nothing seems to work, and I'm in the middle of the semester trying to teach. Any help resolving this would be appreciated.

**Comment by ssp3nc3r** [comment]
I tried again today, and uploaded my id card again, this time, the program rejects me and suggests I should apply as a student:

> Your application cannot be reviewed until you fix the following:
> If you meant to apply as a student, please cancel your application and reapply [here](https://education.github.com/discount_requests/student_application).

The verification is awful and broken. Please help.

**Comment by samus-aran** [comment]
Hi @ssp3nc3r 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐

Here are a few tips to help teachers applying. 

Ensure that your evidence can identify you clearly, text is legible and it can distinguish you as a faculty member. Is there any indication on your evidence that would determine that you are faculty? If there isn't we would suggest taking a photo of another form of evidence along side your ID card. Continue to select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Hope this helps and let us know how you get on 👍

**Reply by ssp3nc3r** [reply]
> Academic Email being at the top

I literally use my academic email address. So, yes. And I provided my University id in the photo. So both of those things you mentioned.
I also provided my university website:  https://sps.columbia.edu/faculty/scott-spencer

**Reply by galk-research** [reply]
I am seeing the same problem.  I have been approved in the past -- twice -- and being a tenured full professor, the situation is unlikely to change...  But in order to get access to co-pilot (e.g., to demonstrate in class!), I apparently need to do this again -- only this time, I get rejected. Using my academic mail.  My faculty ID.  My academic web page.

Very frustrated.

**Reply by umichict** [reply]
I am having the same issue. I had HR send a document to confirm my employment. Github forced me to convert the pdf document into a jpeg and then failed to reverify with the message, "Jpegs" are easy to fake. Why not accept the pdf?

Has anyone had any luck? This has gotten so much worse. Everything is automated, and there is no way to reach a human.

**Reply by gaomy3832** [reply]
I had the same issue for two months by now, and you can see #54126 . Still not working. I am extremely frustrated. Is there any place to complain? Does GitHub care about the customer experience at all?
